### PR TITLE
Document the seed example

### PR DESCRIPTION
**Pull request by dmarticus**
Closes https://github.com/higherkindness/mu-scala-examples/issues/5.

This change updates the readme for `seed` that explains what that example is, it's purpose, and how to run the two apps that within that directory.

### CODE REVIEW

File: seed/client/src/main/resources/application.conf [hunk 1]
```diff
@@ -1,4 +1,3 @@
 name = "SeedClient"
 host = "localhost"
-port = 19683
-port = ${SEED_PORT}
\ No newline at end of file
+port = 19683
```
Comment (cb372) [inline seed/client/src/main/resources/application.conf:3]: Sorry, didn't realise you'd fixed this over here in this PR. I fixed it a slightly different way in #13

File: seed/README.md [hunk 2]
```diff
@@ -1,17 +1,34 @@
...
 ```
 
 Running the client:
 
 ```bash
-sbt seed-client/run
+sbt runProtoClient
 ```
 
 (TODO INCLUDE THE RUN COMMANDS FROM THE BUILD.SBT?)
```
Comment (cb372) [inline seed/README.md:34]: Could we take out this TODO please

File: seed/client/src/main/resources/application.conf (hunk 1 above)
Comment (dmarticus) [reply seed/client/src/main/resources/application.conf:3]: oh yeah the optional env var is probably cleaner; I never saw use using it anywhere but it doesn't hurt to keep it as an option

File: seed/README.md (hunk 2 above)
Comment (dmarticus) [reply seed/README.md:34]: ooops!  yup

**Pull request merged**